scripts/release/rpm/azure-cli.spec

```diff
@@ -5,11 +5,6 @@
 # Turn off python byte compilation
 %global __os_install_post %(echo '%{__os_install_post}' | sed -e 's!/usr/lib[^[:space:]]*/brp-python-bytecompile[[:space:]].*$!!g')
 
-# .el7.centos -> .el7
-%if 0%{?rhel}
-  %define dist .el%{?rhel}
```

Comment:
In RHEL 7.9 `{?dist}` is `.el7_9`.
In CentOS 7, RHEL 8.4, RHEL 9.0.0, it is `.el\d`.